docs/installing.md

```diff
@@ -84,26 +64,29 @@ You can now:
 ansible-playbook -i inventory/hosts setup.yml --tags=start
 ```
 
-Proceed to [Maintaining your setup in the future](#2-maintaining-your-setup-in-the-future) and [Finalize the installation](#3-finalize-the-installation)
-
-
-## 2. Maintaining your setup in the future
+## Finalize the installation
 
-Feel free to **re-run the setup command any time** you think something is off with the server configuration. Ansible will take your configuration and update your server to match.
+Now that services are running, you need to **finalize the installation process** (required for federation to work!) by [Configuring Service Discovery via .well-known](configuring-well-known.md).
 
-Note that if you remove components from `vars.yml`, or if we switch some component from being installed by default to not being installed by default anymore, you'd need to run the setup command with `--tags=setup-all` instead of `--tags=install-all`. See [Playbook tags introduction](#playbook-tags-introduction)
+If you need the base domain (`example.com`) for anything else such as hosting a website, you have to configure it manually, following the procedure described on the linked documentation.
 
-A way to invoke these `ansible-playbook` commands with less typing in the future is to use [just](https://github.com/casey/just) to run them: `just install-all` or `just setup-all`. See [our `justfile`](../justfile) for more information.
+However, if you do not need the base domain for anything else, the easiest way of configuring it is to [serve the base domain](configuring-playbook-base-domain-serving.md) from the integrated web server. It will enable you to use a Matrix user identifier like `@<username>:example.com` while hosting services on a subdomain like `matrix.example.com`.
 
+To configure Service Discovery in this way, add the following configuration to your `inventory/host_vars/matrix.example.com/vars.yml` file:
 
-## 3. Finalize the installation
+```yaml
+matrix_static_files_container_labels_base_domain_enabled: true
+```
 
-Now that services are running, you need to **finalize the installation process** (required for federation to work!) by [Configuring Service Discovery via .well-known](configuring-well-known.md).
+After configuring the playbook, run the installation command:
 
+```sh
+ansible-playbook -i inventory/hosts setup.yml --tags=setup-all,start
```

Comment:
I wonder if this one should suggest `install-all`, instead of `setup-all`. After all, we don't expect that users uninstalled stuff, so the quicker `install-all` tag would be better and more consistent with what we propose above.

```suggestion
ansible-playbook -i inventory/hosts setup.yml --tags=install-all,start
```